dvc/utils/__init__.py

```diff
@@ -41,8 +41,8 @@ def _fobj_md5(fobj, hash_md5, binary, progress_func=None):
             progress_func(len(data))
 
 
-def file_md5(fname, tree=None):
-    """ get the (md5 hexdigest, md5 digest) of a file """
+def file_md5(fname, tree=None, enable_d2u=False):
```

Comment:
Let's create two functions:
```
file_md5
file_dos2unixmd5
```
or
```
get_md5
get_dos2unixmd5
```
just so we don't get tangeled in it ourselves :D

Reply:
Though a flag is good too. Nevermind :slightly_smiling_face: